pepper-apis/src/main/java/org/broadinstitute/ddp/constants/RouteConstants.java

```diff
@@ -161,6 +162,8 @@
         public static final String STUDY_PASSWORD_POLICY = STUDY_DETAIL + "/password-policy";
 
         public static final String STUDY_STATISTICS = STUDY_DETAIL + "/statistics";
+
+        public static final String FILE_UPLOAD_URL = STUDY_DETAIL + "/get-upload-url";
```

Comment:
I think the endpoint URL should just be `/upload-url`. Also, the route should be called `CreateFileUploadUrlRoute` instead -- the word `Get` in the name might be confusing since this is actually a `POST` request.

Reply:
Renamed